README.md

```diff
@@ -166,10 +166,32 @@ And when you are ready to deploy to mainnet, run:
 yarn script:deploy:broadcast
 ```
 
+In addittion, the L1OpUSDCFactory deployment command is:
+```bash
+yarn mainnet:deploy:factory
+```
+
+And when you are ready to deploy to mainnet, run:
+```bash
+yarn mainnet:deploy:factory:broadcast
+```
+
+Alternatively, you can run the deployment scripts over your desired testent by replacing mainnet with testnet in the commands above.
+
 ### Tips For Verifying
 
 - Remember to set the EVM version to `paris` when verifying the contracts.
```

Comment:
I think we should remove the evm version on `foundry.toml` and here put something like "use the default EVM foundry version", I think this will be more future proof. Just an idea, current solution is good.

Reply:
we had issues with it at the moment of verification, I prefer to have it well defined.
Also, in case the EVM version is upgraded and not all the op chains support the new upgrade